inc/dps/private/dps.h

```diff
@@ -170,6 +170,35 @@ void DPS_RxBufferToTx(DPS_RxBuffer* rxBuffer, DPS_TxBuffer* txBuffer);
  */
 void DPS_DumpSubscriptions(DPS_Node* node);
 
+/**
+ * A key store request.
+ */
+struct _DPS_KeyStoreRequest {
+    DPS_KeyStore* keyStore;
+    void* data;
+    DPS_Status (*setKeyAndIdentity)(DPS_KeyStoreRequest* request, const unsigned char* key, size_t keyLen, const unsigned char* id, size_t idLen);
+    DPS_Status (*setKey)(DPS_KeyStoreRequest* request, const unsigned char* key, size_t len);
```

Comment:
nitpicky: I would have used parameter name `keyLen` it is more explicit.  Same for ca len below. 

When is setKeyAndIdentity callback called vs. setKey callback?
Will the setKey callback always be called when setKeyAndIdentity callback is called?